schemas/v0.9.0/tioresult/schema.json

```diff
@@ -4,16 +4,29 @@
 
   "definitions": {
 
+    "output": {
+      "type": "object",
+      "additionalProperties": false,
```

Comment:
@axelsly : I would allow additional properties here, since this will vary based on the model being trained. Then we don't have to specify any required properties.

We can give the whole thing time to calcify (over a few different federated learning models).

Reply:
@axelsly yes, there should be no required properties here and additional properties should be true. all we're doing is specifying that the output be a dictionary that contains any key-value pairs